src/main/target/SITL/target.c

```diff
@@ -79,9 +85,27 @@ int lockMainPID(void) {
 #define RAD2DEG (180.0 / M_PI)
 #define ACC_SCALE (256 / 9.80665)
 #define GYRO_SCALE (16.4)
+
+#define PORT_PWM_RAW 9001 //Out
+#define PORT_PWM 9002     //Out
+#define PORT_STATE 9003   //In
+#define PORT_RC 9004      //In
+
 void sendMotorUpdate() {
     udpSend(&pwmLink, &pwmPkt, sizeof(servo_packet));
 }
+
+int targetParseArgs(int argc, char * argv[]) {
+    //The first argument should be target IP.
+    if (argc > 1) {
+        strcpy(simulator_ip, argv[1]);
```

Comment:
`strncpy(,,sizeof(simulator_ip))`

don't create buffer overflows, ever ;-)

Reply:
Hi there, I am busy these days, maybe will do it later (before the end of this month)

Reply:
@xuhao1 No worries, whenever you have time. Apprechiate your work.